### PR TITLE
27118 - allow links as item-titles

### PR DESCRIPTION
**Pull request by nhaagen**
https://mantis.ilias.de/view.php?id=27118

### CODE REVIEW

**Review by Amstutz — APPROVED** [review]
Thx a lot @nhaagen !

File: src/UI/examples/Item/Standard/with_link_title.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ * With a standard link as title
+ */
+function with_link_title()
+{
+    global $DIC;
+    $f = $DIC->ui()->factory();
+    $renderer = $DIC->ui()->renderer();
+    $app_item = $f->item()->standard($f->link()->standard("ILIAS", "https://www.ilias.de"))
+        ->withProperties(array(
+            "Code Repo" => $f->button()->shy("GitHub", "https://www.github.com"),
```
Comment (klees) [inline src/UI/examples/Item/Standard/with_link_title.php]:
```suggestion
            "Code Repo" => $f->button()->shy("ILIAS on GitHub", "https://www.github.com/ILIAS-eLearning/ILIAS"),
```

**Review by klees — APPROVED** [review]
LGTM, one minor change only.

**Comment by klees** [comment]
@matthiaskunkel This is a rather minor interface change, but I want to announce it on the JF anyway.

**Pull request merged**